fast64_internal/f3d/f3d_material.py

```diff
@@ -4491,6 +4549,7 @@ def mat_register():
 
     Scene.f3dUserPresetsOnly = bpy.props.BoolProperty(name="User Presets Only")
     Scene.f3d_simple = bpy.props.BoolProperty(name="Display Simple", default=True)
+    Scene.dont_ask_colorspace = bpy.props.BoolProperty(name="Don't ask again")
```

Comment:
Can you move this prop to Fast64_Properties (so access like scene.fast64.dont_ask_colorspace)